GDJS/Runtime/scenestack.ts

```diff
@@ -70,6 +70,44 @@ namespace gdjs {
         } else {
           logger.error('Unrecognized change in scene stack: ' + request);
         }
+
+        const allSyncData: GameSaveState = JSON.parse(
+          localStorage.getItem('save') as string
+        );
+
+        if (currentScene._isLoadingRequested && allSyncData) {
+          currentScene
+            .getGame()
+            .updateFromNetworkSyncData(allSyncData.gameNetworkSyncData, {
+              skipMultiplayerInstructions: true,
+            });
+          currentScene
+            .getGame()
```

Comment:
You're using currentScen.getGame() 3 times in this function. That should ring a bell :)